src/routes/(console)/project-[region]-[project]/databases/database-[database]/table-[table]/createRecord.svelte

```diff
@@ -26,6 +27,8 @@
 
     let isSubmitting = $state(false);
     let columnFormWrapper: HTMLDivElement | null = $state(null);
+    let createMore = $state(false);
+    let formResetKey = $state(0);
```

Comment:
can we not use the `createRow` variable as a `#key`? The `formResetKey += 1` isn't that nice.